pyproject.toml

```diff
@@ -19,12 +19,13 @@ httpx = "^0.24.1"
 pydantic = "1.10.11"
 pefile = "^2023.2.7"
 uvicorn = { extras = ["standard"], version = "^0.23.1" }
-python-magic = "^0.4.27"
 beanie = "^1.19.2"
 typer = { extras = ["all"], version = "^0.9.0" }
 jinja2 = "^3.1.2"
 pytest = "^7.4.0"
 python-multipart = "^0.0.6"
+file-magic = "^0.4.1"
```

Comment:
Why did you add this import without using it? Technically it has a namespace collision with our other dependency of `python-magic`. Please choose one or the other.